FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Frank Carnevale on 8/23/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedItemsMapper {
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly.

It was written explicitly in the first lectures of the program for teaching purposes, but you don't need to do it in your code.

Please, remove all explicit `internal` keywords.
```suggestion
final class FeedItemsMapper {
```

Reply:
Done. Thanks!